data/foods/pizzaToppings.json

```diff
@@ -4,11 +4,16 @@
     "anchovies",
     "artichoke",
     "bacon",
+    "breakfast bacon",
+    "canadian bacon",
```

Comment:
@webmaven How about "Canadian bacon"? For people who want everything in lowercase they can change it themselves, but it's difficult to add caps when there are none in the source data to begin with.